e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -262,6 +263,27 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
     popover().contains("80.36");
   });
 
+  it("should show an info card filter columns in the popover", () => {
```

Comment:
I think in this case there is no real BE interaction involved, we could add a FE unit test

Reply:
I've added a unit test too. Would you have me remove the e2e test?